doc/source/_toc.yml

```diff
@@ -325,9 +332,10 @@ parts:
               - file: workflows/advanced
               - file: workflows/api/api
 
-      - file: cluster/getting-started
-        title: "Ray Clusters"
+      - title: "Ray Clusters"
         sections:
+          - file: cluster/getting-started
+            title: "Ray Clusters overview"
```

Comment:
```suggestion
            title: "What are Ray Clusters?"
```

Reply:
Any particular reason why the Clusters landing page doesn't follow the same pattern with "What is/are Ray Clusters"? If not, I would recommend being consistent.